.github/workflows/developer_onboarding_notification.yml

```
@@ -0,0 +1,64 @@
+
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/merge
+      - name: Count merged PRs
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const author = context.payload.pull_request.user.login;
+            const repo = context.repo.repo;
+            const owner = context.repo.owner;
+            
+            const { data } = await github.rest.search.issuesAndPullRequests({
+              q: `repo:${owner}/${repo} type:pr state:closed author:${author}`
+            });
+            const prCount = data.items.filter(pr => pr.pull_request.merged_at).length;
+            
+            core.exportVariable('PR_COUNT', prCount);
+            core.exportVariable('AUTHOR', author);
+      - name: Comment on the PR
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const prCount = parseInt(process.env.PR_COUNT);
+            const author = process.env.AUTHOR;
+            const mention = '@HardikGoyal2003';
```

```suggestion
            const author = process.env.AUTHOR;
            const mention = 'HardikGoyal2003';
```

To be consistent with `author`

---

Done